db/schema.rb

```diff
@@ -1,4 +1,3 @@
-# encoding: UTF-8
```

Comment:
I think I forgot to run migrations after updating to Rails 5 earlier this week :(